relay-event-normalization/src/event.rs

```diff
@@ -2515,4 +2548,75 @@ mod tests {
 
         assert_eq!(user.other, Object::new());
     }
+
```

Comment:
Are there any tests for `normalize_exceptions` function which runs on the entire `Event`?  So we could cover the entire normalization step. I can't see any. 

Can some be added as well in this PR?

Reply:
Yep yep, added in [7738516](https://github.com/getsentry/relay/pull/3072/commits/7738516faf7616524a873160509feb826a629a5c).